modules/library/src/main/scala/zio/elasticsearch/ElasticRequest.scala

```diff
@@ -645,7 +650,13 @@ object ElasticRequest {
             })
         }
 
-      fromJson merge sizeJson merge highlightsJson merge sortJson merge self.query.toJson merge searchAfterJson merge sourceJson
+      fromJson merge
+        sizeJson merge
+        highlightsJson merge
+        sortJson merge
+        Obj("query" -> query.toJson(fieldPath = None)) merge
```

Comment:
If it makes no difference, stylistically I would put `Obj("query" -> query.toJson(fieldPath = None))` on first place.